example/vm/config.yaml

```diff
@@ -0,0 +1,201 @@
+# Last Collector-Contrib Validation: v0.96.0
+receivers:
+    prometheus/self:
+        config:
+            scrape_configs:
+              - job_name: otel-collector
+                scrape_interval: 5s
+                static_configs:
+                - labels:
+                    collector_name: sn-collector
+                  targets:
+                    - 0.0.0.0:8888
+    hostmetrics:
+    # Hostmetrics scrapes metrics from various host systems.
+    # For more details, see
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md
+        collection_interval: "30s"
+        root_path: /hostfs
+        scrapers:
+            cpu:
+                metrics:
+                    system.cpu.utilization:
+                        enabled: true
+            disk: {}
+            # cpu and disk are only supported if build with CGO_ENABLED=1
+            load: {}
+            filesystem:
+                metrics:
+                    system.filesystem.utilization:
+                        enabled: true
+                exclude_mount_points:
+                    match_type: regexp
+                    mount_points:
+                        - /dev/.*
+                        - /proc/.*
+                        - /sys/.*
+                        - /run/k3s/containerd/.*
+                        - /var/lib/docker/.*
+                        - /var/lib/kubelet/.*
+                        - /snap/.*
+                exclude_fs_types:
+                    match_type: strict
+                    fs_types:
+                        - autofs
+                        - binfmt_misc
+                        - bpf
+                        - cgroup2
+                        - configfs
+                        - debugfs
+                        - devpts
+                        - devtmpfs
+                        - fusectl
+                        - hugetlbfs
+                        - iso9660
+                        - mqueue
+                        - nsfs
+                        - overlay
+                        - proc
+                        - procfs
+                        - pstore
+                        - rpc_pipefs
+                        - securityfs
+                        - selinuxfs
+                        - squashfs
+                        - sysfs
+                        - tracefs
+            memory:
+                metrics:
+                    system.memory.utilization:
+                        enabled: true
+            # paging:
+            # processes:
+            # process:
+            network: {}
+    otlp:
+    # OTLP is the default protocol supported by all language implementations.
+    # For more details, see
+    # https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/README.md
+        protocols:
+            grpc:
+                endpoint:
+            http:
+                endpoint:
+processors:
```

Comment:
I noticed the otel-cloud-stack chart includes a [resourcedetection/env](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor) receiver. Does that apply here too / should we include it? https://github.com/lightstep/otel-collector-charts/blob/main/charts/otel-cloud-stack/values.yaml#L398-L401

Reply:
yeah this is worth having